lightning/src/ln/channelmonitor.rs

```diff
@@ -2396,9 +2396,9 @@ impl ChannelMonitor {
 			// Scan all input to verify is one of the outpoint spent is of interest for us
 			let mut claimed_outputs_material = Vec::new();
 			for inp in &tx.input {
-				if let Some(ancestor_claimable_txid) = self.claimable_outpoints.get(&inp.previous_output) {
+				if let Some(first_claim_txid_height) = self.claimable_outpoints.get(&inp.previous_output) {
```

Comment:
Why first_claim_txid_height?, it's still `Sha256dHash` there?

Reply:
its a pair - .0 is a Sha256dHash, .1 is the height at which it was added.

Reply:
`first_claim_txid_and_height` but nevermind :p